components/network/inputs-default.tf

```diff
@@ -93,4 +93,20 @@ variable "product" {
 
 variable "additional_routes_appgw" {
   default = []
+}
+
+variable "route_name" {
+  default = "default"
+}
+
+variable "route_address_prefix" {
```

Comment:
Is this the required route setup?

Example of route table you are looking to automate
https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourceGroups/core-infra-aat/providers/Microsoft.Network/routeTables/aks-aat-core-infra-route-table/overview

Reply:
Resolved